lua/lspconfig/server_configurations/denols.lua

```diff
@@ -18,19 +18,21 @@ local function virtual_text_document_handler(uri, result)
   end
 
   for client_id, res in pairs(result) do
-    local lines = vim.split(res.result, '\n')
-    local bufnr = vim.uri_to_bufnr(uri)
+    if res.result ~= nil then
```

Comment:
- a 1-line comment is worth adding here to give a hint about the reasoning.
- should probably log at warn or info level

```suggestion
    -- May be nil because of race, deno server will eventually send a result. #1995
    if res.result == nil then
      log.warn(...)
    else
```

Reply:
I can't find log module, where do I get it from?

Reply:
require('vim.lsp.log').warn('foo')

you can find the result in : 

    exe 'e' stdpath('log')

Reply:
done